README.md

```diff
@@ -134,13 +142,10 @@ end
 
 (See the [Mix Release Config Options](#mix-release-config-options) for additional options)
 
-3. To build a release for the platforms defined in your `mix.exs` file: `MIX_ENV=prod mix release`
-4. You can also override the target platforms using the `BURRITO_TARGET` environment variable
-  * To build a release for Windows: `MIX_ENV=prod BURRITO_TARGET=win64 mix release`
-  * To build a release for MacOS: `MIX_ENV=prod BURRITO_TARGET=darwin mix release`
-  * To build a release for Linux: `MIX_ENV=prod BURRITO_TARGET=linux mix release`
+3. To build a release for all the targets defined in your `mix.exs` file: `MIX_ENV=prod mix release`
+4. You can also do one-shot builds by setting the `BURRITO_TARGET` environment variable to a target's alias (ex. `BURRITO_TARGET=macos` would only build the `macos` target we defined above.)
```

Comment:
The meaning of "one-shot builds" isn't obvious to me. Would it be clearer to say something like this?

> You can also build a single target by setting the `BURRITO_TARGET` environment variable to the alias for that target (e.g. Setting `BURRITO_TARGET=macos` builds only the `macos` target defined above.)

Also minor nitpick: `ex.` is sometimes used to mean "example", but more traditionally abbreviates "exercise", and `e.g.` would be the more common abbreviation here.

Reply:
I like that wording a lot more!